compose.yaml

```diff
@@ -0,0 +1,11 @@
+services:
+  api:
+    build:
+      context: .
+    command: ["python", "-m", "torch_ecg_volta.api.server"]
```

Comment:
How is python aware of the dependencies downloaded by poetry in the build step ?